src/backend/controllers/tankController.js

```diff
@@ -80,6 +126,50 @@ exports.favoriteTank = async (req: Request, res: Response) => {
 	});
```

Comment:
If we call favoritTank, it needs to handle unfavorite'ing the team, right?

Likewise, if we favorite a team, we need to unfavorite the individual favorite tank, right?

Reply:
we talked about this offline and decided that it is fine to support having both a 1v1 and 3v3 wager